util/util.go

```diff
@@ -83,11 +87,35 @@ func SaveWebauthnSession(session *sessions.Session, key string, sessionData *web
 	return nil
 }
 
+func RandInit() {
+	var b [8]byte
+	_, err := crypto_rand.Read(b[:])
+	if err != nil {
+		panic("cannot seed math/rand package with cryptographically secure random number generator")
+	}
+	math_rand.Seed(int64(binary.LittleEndian.Uint64(b[:])))
+}
+
+func genChallenge(int len) {
+	if len < 32 {
+		//overide for minimum allowable value, we want to be able to set way beyond but enforce at least 32 bytes
+		len = 32
+	}
+	//spec recomends 16 bytes challenge, we're going to double that
+	challenge := make([]byte, len)
+	_, err := crypto_rand.Read(challenge)
+	if err != nil {
+		panic("failed to seed challenge from crypto/rand cryptographically secure function")
+	}
+	r := base64.RawURLEncoding.EncodeToString(challenge)
+	return r
+}
+
 // Generate a random string of alpha characters of length n
 func RandStringBytesRmndr(n int) []byte {
```

Comment:
As with `RandInit()` this _may_ be able to be removed.

Reply:
Removed within #[f7f4776](https://github.com/Quiq/webauthn_proxy/pull/3/commits/f7f47768feb76d64a577beefdc78b2e580702616)